lib/system.g

```diff
@@ -571,3 +573,8 @@ end);
 ##  </ManSection>
 ##
 #T really ???
+
+if GAPInfo.CommandLineOptions.systemfile <> "" and
+   IsReadableFile( GAPInfo.CommandLineOptions.systemfile ) then
+  READ( GAPInfo.CommandLineOptions.systemfile );
```

Comment:
```suggestion
if GAPInfo.CommandLineOptions.systemfile <> "" and not READ( GAPInfo.CommandLineOptions.systemfile ) then
    Error( "file \"", name, "\" must exist and be readable" );
```

In view of what @ChrisJefferson suggested (and I agree), how about doing something like the above?